pkg/spec3/response.go

```diff
@@ -78,20 +78,29 @@ func (r ResponsesProps) MarshalJSON() ([]byte, error) {
 
 // UnmarshalJSON unmarshals responses from JSON
 func (r *ResponsesProps) UnmarshalJSON(data []byte) error {
-	var res map[string]*Response
+	var res map[string]json.RawMessage
```

Comment:
This change will be removed here and included in a separate PR: https://github.com/kubernetes/kube-openapi/pull/372